lib/src/delegates/asset_picker_builder_delegate.dart

```diff
@@ -491,7 +494,7 @@ class DefaultAssetPickerBuilderDelegate
                         child: Column(
                           children: <Widget>[
                             Expanded(child: assetsGridBuilder(context)),
-                            if (!isSingleAssetMode) bottomActionBar(context),
+                            if (!isSingleAssetMode && specialPickerType != SpecialPickerType.noPreview) bottomActionBar(context),
```

Comment:
I think we can make the `specialPickerType != SpecialPickerType.noPreview` as `bool get isPreviewDisabled`, without hard coded in multiple places.

Reply:
Yea, I thought about that too. Done.